content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/moves_history.rst

```diff
@@ -0,0 +1,100 @@
+=======================
+Moves history dashboard
+=======================
+
+The *Moves History* report in Odoo *Inventory* provides a detailed record of product movements
+(containing past and current locations), lot numbers, and reasons for movement. Reports can be
+generated for any time frame, making this report essential for analyzing stock levels, monitoring
+inventory turnover, and identifying any discrepancies in inventory.
+
+.. note::
+   The reporting feature is only accessible to users with :doc:`admin access
+   <../../../../general/users/access_rights>`.
+
+To access the stock report, go to :menuselection:`Inventory app --> Reporting --> Moves History`.
+
+.. image:: moves_history/moves-history.png
+   :align: center
+   :alt: Display Moves History report accessible from Inventory > Reporting > Moves History.
```

Comment:
No need to reiterate the previous instruction in an alt-tag, rather it should just describe what is depicted in the image :)
```suggestion
   :alt: Display Moves History report.
```